docs/recipes/ui/add-routing.md

```diff
@@ -0,0 +1,115 @@
+﻿# How do I add routes to a SAFE app?
+
+When building larger apps, you probably want different pages to be accessible through different URLs. In this recipe, we show you how to add routes to different pages to an application, including adding a "page not found" page that is displayed when an unknown URL is entered.
+
+## 1. Adding the Feliz router
+
+Install Feliz.Router in the client project
+
+```bash
+dotnet paket add Feliz.Router -p Client
+```
+
+To include the router in the Client, add `open Feliz.Router` at the top of Index.fs
+
+## 2. Adding the URL object
+
+Add the current page to the model of the client, using a new `Page` type
+
+```fsharp
+type Page =
+    | TodoList
+    | NotFound
+
+type Model =
+    { CurrentPage: Page
+      Todos: Todo list
+      Input: string }
+```
+
+## 3.  Parsing URLs
+
+Create a function to parse an url to a page, including a wildcard for unmapped pages
```

Comment:
```suggestion
Create a function to parse a URL to a page, including a wildcard for unmapped pages
```

Assuming you're pronouncing URL "you are ell", "an" is more appropriate than "a".